bpf/process/types/basic.h

```diff
@@ -1009,7 +1012,17 @@ static inline __attribute__((always_inline)) int match_binaries(void *sel_names,
 }
 
 static inline __attribute__((always_inline)) int
-selector_arg_offset(__u8 *f, struct msg_generic_kprobe *e, __u32 selidx)
+generic_process_filter_binary(struct event_config *config)
+{
+	/* single flag bit at the moment (FLAGS_EARLY_FILTER) */
+	if (!config->flags)
```

Comment:
As I believe that we will need `flags` later as well (and add more `FLAGS_*`), why not `(!(config->flags & FLAGS_EARLY_FILTER))` instead of `(!config->flags)`? In that case we can also remove the comment as well.

Reply:
yea, it was a shortcut because flags is now 0 or 1, but I'll change it to proper check